### PR TITLE
CONEKTA:  Fix remote test

### PR DESCRIPTION
**Pull request by javierpedrozaing**
Description
-------------------------
We need to check if all remote test are passing successfully
Updating test cards' year and removing unusual test

JIRA ticket number
-------------------------
[GWI-141](https://spreedly.atlassian.net/browse/GWI-141)

Unit test
-------------------------
Finished in 0.011947 seconds.

13 tests, 85 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

1088.14 tests/s, 7114.76 assertions/s

Remote test
-------------------------
Finished in 54.175181 seconds.

15 tests, 56 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

0.28 tests/s, 1.03 assertions/s

Rubocop
-------------------------
739 files inspected, no offenses detected

### CODE REVIEW

**Review by Heavyblade — APPROVED** [review]
Looks OK, just leave an small comment on test name

File: test/remote/gateways/remote_conekta_test.rb [hunk 1]
```diff
@@ -61,6 +61,11 @@ def test_successful_purchase_with_installments
     assert_equal nil, response.message
   end
 
+  def test_unsuccessful_purchase_with_not_supported_currency
```
[the remark this answers is not in the document]
Comment (javierpedrozaing) [reply test/remote/gateways/remote_conekta_test.rb:64]: Oh., Thanks, but instead update the test title, I have removed the parameter monthly_installments, bacuase the idea is show what is the behavior o response when you are using one not supported currency.

File: test/remote/gateways/remote_conekta_test.rb [hunk 2]
```diff
@@ -9,8 +9,8 @@ def setup
     @credit_card = ActiveMerchant::Billing::CreditCard.new(
       number:             '4242424242424242',
       verification_value: '183',
-      month:              '01',
-      year:               '2019',
+      month:              '12',
+      year:               '2024',
```
Comment (curiousepic) [inline test/remote/gateways/remote_conekta_test.rb:13]: Can we make this dynamic?

Comment (javierpedrozaing) [reply test/remote/gateways/remote_conekta_test.rb]: done

File: test/remote/gateways/remote_conekta_test.rb [hunk 3]
```diff
@@ -1,4 +1,5 @@
 require 'test_helper'
+require 'date'
```
Comment (curiousepic) [inline test/remote/gateways/remote_conekta_test.rb:2]: FYI we don't actually need to require it here; i'll remove it on merge

**Review by curiousepic — APPROVED** [review]
Looks good, thanks!

**Pull request closed without merging**